sale_project_milestone/models/sale_order.py

```diff
@@ -0,0 +1,18 @@
+# Copyright 2023 - today Numigi (tm) and all its contributors (https://bit.ly/numigiens)
+# License LGPL-3.0 or later (http://www.gnu.org/licenses/lgpl).
+
+from odoo import models
+
+
+class SaleOrder(models.Model):
+
+    _inherit = "sale.order"
+
+    def action_view_milestone(self):
```

Comment:
@lanto-razafindrabe 
Action already existe in sale_project module, see [here](https://github.com/odoo/odoo/blob/16.0/addons/sale_project/models/sale_order.py#L174)